packages/alert/src/Alert.tsx

```diff
@@ -25,57 +20,61 @@ export enum AlertTypes {
   BANNER = 'banner',
 }
 
-export type BaseAlertProps = {
-  /**
-   * The description of the component.
-   */
-  description: string
+export enum AlertVariant {
+  CRITICAL = 'critical',
+  INFORMATIVE = 'informative',
+  SUCCESS = 'success',
+  WARNING = 'warning',
+}
 
+export type BaseAlertProps = PropsWithChildren<{
   /**
    * The title of the component.
    */
   title?: string
 
   /**
-   * The action link of the component.
-   */
-  actionLinkUrl?: string
-
-  /**
-   * The action link text of the component.
+   * The variant to use. Defaults to 'success'.
+   * @defaultValue 'success'
    */
-  actionLinkText?: string
+  variant?: AlertVariant
 
   /**
-   * The variant to use. Defaults to "success".
+   * The variant to use. Defaults to 'relative'.
+   * @defaultValue 'relative'
    */
-  variant?: 'success' | 'warning' | 'critical' | 'informative'
+  type?: AlertTypes
 
   /**
-   * The variant to use. Defaults to "relative".
+   * Show / Hide close button. Defaults to true.
+   * @defaultValue true
    */
-  type?: AlertTypes
+  showCloseButton?: boolean
 
   /**
-   * Show / Hide close button
+   * Close button title. Defaults to 'Close'.
+   * @defaultValue 'Close'
    */
-  showCloseButton?: boolean
+  closeButtonTitle?: string
```

Comment:
It might be better to rename this to `closeButtonText`, `Title` sorts of implies a text hierarchy

Reply:
@Zaggen I selected the name `closeButtonTitle` because the value of this prop is set as the value of close button `title` attribute (meaning this value is displayed as a tooltip)